apps/intellij-extension/settings.gradle.kts

```diff
@@ -0,0 +1,8 @@
+pluginManagement {
+    repositories {
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+
+rootProject.name = "otter-extension"
```

Comment:
Could we have a linter to have same rule as ts file with empty line at the end of the file

Reply:
I'll be the linter for this time :)